what_is.md

```diff
@@ -222,10 +237,11 @@ tree sequence over geographical space.
 
 
 A major benefit of "tree sequence thinking" is the close relationship between the
-tree sequence and the underlying biological processes that produced
-the genomes in the first place. For example, each branch point in one of the trees above
-represents a most recent common ancestor (MRCA), in other words a genome which existed at
-a specific time in the past. 
+tree sequence and the underlying biological processes that produced the genetic sequences
+in the first place. For example, each branch point in one of the trees above
```

Comment:
worth adding to start introducing the terminology?
```suggestion
in the first place. For example, each branch point (node) in one of the trees above
```

Reply:
(or "internal node")?